docs/user/installation.rst

```diff
@@ -40,7 +34,7 @@ Here is an example using a Python 3.5 interpreter:
   requests-jwt==0.4
   shaarli-client==0.1.0
 
-Get the source code
+From the source code
```

Comment:
there must be as many underlining dashes as there are characters in a title (yup, this is one of the few annoying sides of rst)